cpp/ql/test/query-tests/Security/CWE/CWE-190/semmle/tainted/test2.cpp

```diff
@@ -0,0 +1,28 @@
+
+typedef signed long long int s64;
```

Comment:
I forgot to simplify this.  The type doesn't matter, I get the same results with `int`.